dropwizard-health/src/main/java/io/dropwizard/health/response/HealthResponse.java

```diff
@@ -1,57 +1,67 @@
 package io.dropwizard.health.response;
 
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
+import io.dropwizard.health.HealthStateView;
+import java.util.Collection;
 import java.util.Objects;
-import java.util.Optional;
+import javax.annotation.Nonnull;
 
-public abstract class HealthResponse {
-    protected final boolean healthy;
-    @Nullable
-    protected final String message;
+public final class HealthResponse {
+    private final boolean healthy;
+    @Nonnull
+    private final String message;
+    @Nonnull
+    private final String contentType;
     @Nonnull
-    protected final String contentType;
+    private final Collection<HealthStateView> views;
```

Comment:
I don't see why `views` is a member of this. I see `HealthResponse` as an object that is passed from the HealthResponseProvider to a HealthResponder for the purposes of turning it into a materialized response. At this point, the responder should just be worried about how to turn the `message`, `contentType`, and healthy flag into something to send out over the wire (i.e. into http or whatever).